docs/components/lists.html

```diff
@@ -0,0 +1,355 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link
+			rel="stylesheet"
+			href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.4.0/styles/night-owl.min.css"
+			integrity="sha512-i5X6Fdn/ZqvGSqPrdMa3FgcpXM/Nr6YccSKFYT93zljl/HZDEpvBbE5Pxp91eiWGccZLrL/LDQJd7fjTRYsVaA=="
+			crossorigin="anonymous"
+			referrerpolicy="no-referrer"
+	/>
+    <link rel="stylesheet" href="../../css/base-layout.css">
+    <link rel="stylesheet" href="../../css/main.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet">
+    <title>Playground</title>
+</head>
+<body class="holy-grail-layout grid">
+
+    <!-- Top Nav Bar  -->
+    <header class="top-nav children-centered">
+        <section class="brand-logo">
+            <div class="brand-img">
+                <img src="../../assets/ingredient_logo.svg" alt="ingredient-logo">
+            </div>
+            <div class="txt-lg">Ingredient.</div>
+        </section>
+        <section class="nav-links pd10">
+            <a href="../../index.html" class="nav-item">Home</a>
+            <a href="../intro.html" class="nav-item">Docs</a>
+            <a href="https://github.com/coldpigli/Ingredient/" class="nav-item">Contribute</a>
+        </section>
+    </header>
+
+    <!-- SideBar  -->
+    <aside class="side-nav flex-vertical shadow-all">
+         <h1 class="txt-lg gap-u20 gap-d20">v1.0</h1>
+        </section>
+        <section class="side-nav-group gap-d20">
+            <h4 class="sub-title gap-d20">INTRODUCTION</h4>
+            <div class="sub-list">
+                <ul>
+                    <li class="sub-list-items">
+                        <span class="material-icons md-24">flag</span>
+                        <a href="../intro.html">Get Started</a>
+                    </li>
+                    <li class="sub-list-items">
+                        <span class="material-icons md-24">palette</span>
+                        <a href="../color-page.html">Colors</a>
+                    </li>
+                    <li class="sub-list-items">
+                        <span class="material-icons md-24">format_size</span>
+                        <a href="../typography-page.html">Typography</a>
+                    </li>
+                    <li class="sub-list-items">
+                        <span class="material-icons md-24">fast_forward</span>
+                        <a href="../way-forward-page.html">Way Forward</a>
+                    </li>
+                </ul>
+            </div>
+        </section>
+        <section class="side-nav-group gap-d20">
+            <h4 class="sub-title gap-d20">COMPONENTS</h4>
+            <div class="sub-list">
+                <ul>
+                    <li class="sub-list-items">
```

Comment:
as I see you used sub-list item class in every list tag  you can also use  one class  for list for this particular div

Reply:
Will look further into this. Thanks